docs/application/native/guides/connectivity/inm.md

```diff
@@ -0,0 +1,171 @@
+# INM
+
+
+INM means Intelligent Network Monitoring. It can be used to get information about the network provided by well-known linux functions. Some features provided by this API are: detecting IP collision, dumping network status, dumping TCP, monitoring ethernet, monitoring Wi-Fi module state and getting network statistics.
+
+This feature is supported in mobile, tv and wearable profile.
```

Comment:
Kindly use full form first with short form in brackets to maintain consistency across pages. Also suggesting to use bullet points to list all API features, that will give a good presentation and reading flow to the page. A doubt here that “can be” sounds ambiguous, are there some more ways to the same? Or can we use “…is used to get information…”

Suggesting rephrase as:

Intelligent Network Monitoring (INM) API is used to get information about the network provided by Linux functions. INM is supported by mobile, TV, and wearable profile. Following are the features provided by INM API:

- Detecting IP collision
- Dumping network status
- Dumping TCP
- Monitoring Ethernet
- Monitoring Wi-Fi module state
- Getting network statistics

Kindly verify the suggestions for technical accuracy, otherwise feel free to discuss.

Reply:
Thank you for the suggestion, has been added to the next commit.